packages/xo-web/src/common/intl/messages.js

```diff
@@ -1317,6 +1317,9 @@ const messages = {
   availableTemplateVarsTitle: 'Available template variables',
   templateNameInfo: 'the VM\'s name. It must not contain "_"',
   templateIndexInfo: "the VM's index, it will take 0 in case of single VM",
+  templateAntiSlash: 'will be replaced by "\\"',
+  templateEscape:
+    'used to escape the template variables, e.g: "\\%" will be replaced by "%" instead of the VM\'s index',
```

Comment:
I don't think we should add these to the list since they're not variables. Maybe a simple tip (with the info icon) that say "Escape any variable with a preceding backslash (\\)". If you know what "escaping" means in the first place, then it's more than enough.